src/Sarif/Visitors/RebaseUriVisitor.cs

```diff
@@ -64,38 +85,75 @@ public override PhysicalLocation VisitPhysicalLocation(PhysicalLocation node)
 
         public override Run VisitRun(Run node)
         {
+            _files = node.Files;
+
             Run newRun = base.VisitRun(node);
 
-            if (newRun.Files != null)
-            {
-                FixFiles(newRun);
-            }
-            
+            newRun.Files = _files;
+
             if (node.Properties == null)
```

Comment:
If OriginalUriBaseIds is now a first class object, we don't need to touch the propertybag here.